failures when running tests using python 3.6 on windows

The following failures are seen on python 3.6

```python
======================================================================
FAIL: test_simple_from_running_python (okonomiyaki.runtimes.tests.test_runtime.TestPythonRuntime)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\itziakos\Projects\okonomiyaki\okonomiyaki\runtimes\tests\test_runtime.py", line 34, in test_simple_from_running_python
    os.path.realpath(NORM_EXECUTABLE).lower()
AssertionError: 'c:\\users\\itziakos\\.edm\\envs\\okonomiyaki-dev-3\\python3.exe' != 'c:\\users\\itziakos\\.edm\\envs\\okonomiyaki-dev-3\\python.exe'
- c:\users\itziakos\.edm\envs\okonomiyaki-dev-3\python3.exe
?                                                     -
+ c:\users\itziakos\.edm\envs\okonomiyaki-dev-3\python.exe


----------------------------------------------------------------------
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,6 +5,7 @@ matrix:
     include:
         - python: 2.7
         - python: 3.5
+        - python: 3.6
         - python: pypy
         - python: 2.7
           env: RUN_FLAKE8=1

--- a/.travis.yml
+++ b/.travis.yml
@@ -24,7 +24,7 @@ branches:
     - maint/*
 
 install:
-  - pip install -U pip wheel coveralls;
+  - pip install -q -U pip wheel coveralls;
   - pip install -q .
   - pip install -q -rdev_requirements.txt
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -24,10 +24,11 @@ init:
   - ps: $Env:path = $Env:python + ";" + $Env:python + "\scripts;" + $Env:path
 
 install:
-  - ps: python -m pip install --upgrade pip wheel coveralls
+  - ps: python -m pip install --upgrade pip wheel
+  - ps: pip install coveralls
   - ps: pip --version
   - ps: pip install .
-  - ps: pip install dev_requirements.txt
+  - ps: pip install -r dev_requirements.txt
 test_script:
   - cmd: coverage run -m haas okonomiyaki
 on_success:

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -25,11 +25,8 @@ init:
 
 install:
   - ps: python -m pip install --upgrade pip wheel
-  - ps: pip install coveralls
   - ps: pip --version
   - ps: pip install .
   - ps: pip install -r dev_requirements.txt
 test_script:
   - cmd: coverage run -m haas okonomiyaki
-on_success:
-  - coveralls

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -20,6 +20,9 @@ environment:
 cache:
   - C:\Users\appveyor\AppData\Local\pip\Cache
 
+matrix:
+  fast_finish: true
+
 init:
   - ps: $Env:path = $Env:python + ";" + $Env:python + "\scripts;" + $Env:path
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,0 +1,34 @@
+build: false
+skip_branch_with_pr: true
+# Actual shallow *clone*. The shallow_clone appveyor option downloads
+# an archive from GitHub API and does *not* clone!
+platform: x64
+
+environment:
+
+  global:
+    PYTHONUNBUFFERED: "1"
+
+  matrix:
+    - python: "C:/Python27-x64"
+    - python: "C:/Python27"
+    - python: "C:/Python35-x64"
+    - python: "C:/Python35"
+    - python: "C:/Python36-x64"
+    - python: "C:/Python36"
+
+cache:
+  - C:\Users\appveyor\AppData\Local\pip\Cache
+
+init:
+  - ps: $Env:path = $Env:python + ";" + $Env:python + "\scripts;" + $Env:path
+
+install:
+  - ps: python -m pip install --upgrade pip wheel coveralls
+  - ps: pip --version
+  - ps: pip install .
+  - ps: pip install dev_requirements.txt
+test_script:
+  - cmd: coverage run -m haas okonomiyaki
+on_success:
+  - coveralls

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,7 +1,5 @@
 build: false
 skip_branch_with_pr: true
-# Actual shallow *clone*. The shallow_clone appveyor option downloads
-# an archive from GitHub API and does *not* clone!
 platform: x64
 
 environment:

--- a/okonomiyaki/runtimes/runtime.py
+++ b/okonomiyaki/runtimes/runtime.py
@@ -98,7 +98,7 @@ class PythonRuntime(Runtime):
             scriptsdir = bindir = posixpath.join(prefix, "bin")
             paths = (bindir, )
 
-        if version.major == 3:
+        if version.major == 3 and platform.os_kind != OSKind.windows:
             executable = u"python3"
         else:
             executable = u"python"

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py35,pypy,flake8
+envlist = py27,py35,py36,pypy,flake8
 
 [testenv]
 deps= -rdev_requirements.txt

